ludwig/utils/loss_utils.py

```diff
@@ -3,7 +3,10 @@
 
 def rmspe_loss(targets: torch.Tensor, predictions: torch.Tensor) -> torch.Tensor:
     """Root mean square percentage error."""
```

Comment:
Let's update this comment to explain the expected behavior when there are targets that contain 0, perhaps adapted from [scikit-learn](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html)


> MAPE output is non-negative floating point. The best value is 0.0. But note that bad predictions can lead to arbitrarily large MAPE values, especially if some y_true values are very close to zero. Note that we return a large value instead of inf when y_true is zero.